articles/article-116-esoteric-programming-languages.md

```diff
@@ -0,0 +1,33 @@
+<!-- BEGIN TITLE -->
+# Esoteric Science
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+There are thousands of programming languages. But some of them are just laborious and inefficient. They are not designed for practical usage but to implement different aspects of unusual language concepts. Those programming languages are called esoteric. One of them is called Brainfuck. It consists of the eight language commands:
```

Comment:
I would expand a bit on the first two sentences:
Since the advent of computers, thousands of programming languages have been invented. While some are great and are still used today, other languages are laborious and inefficient.

Reply:
might want to change the word esoteric to  esoteric programming language (sometimes shortened to esolang)

Reply:
Changed the first two sentences